libs/sdk/e2e-schematics/src/generators/configure-percy/index.spec.ts

```diff
@@ -1,71 +1,48 @@
 import { cypressProjectGenerator } from '@nrwl/cypress';
-import { createTreeWithEmptyV1Workspace } from '@nrwl/devkit/testing';
-
-import { updateJson } from '../../utils';
+import { createTreeWithEmptyWorkspace } from '@nrwl/devkit/testing';
 
 import configurePercy from './index';
 
 describe('configure-percy', () => {
   it('should import percy', async () => {
-    const tree = createTreeWithEmptyV1Workspace();
+    const tree = createTreeWithEmptyWorkspace();
```

Comment:
Switching to `createTreeWithEmptyWorkspace` creates an NX workspace with `workspace.json` and separate `project.json` files rather than using `angular.json`. It doesn't affect how the test works, but this better reflects how our workspace is set up.

Reply:
Nice!